packages/shared/src/components/modals/RanksModal/index.tsx

```diff
@@ -44,7 +44,7 @@ export default function RanksModal({
           previousRank={previousRank}
         />
         {!user && (
-          <div className="flex flex-col items-center mt-2">
+          <div className="flex flex-col items-center mt-2 mb-3">
```

Comment:
```suggestion
          <div className="flex flex-col items-center mt-2 mb-7">
```

Reply:
@rebelchris Thanks for the feedback. I've updated the spacing. 😊